### ISSUE TITLE
Data Payload hanging

### ISSUE THREAD

**Issue by alexiswl** [umccr/orca-ui#71]
![image](https://github.com/user-attachments/assets/fc8ee48f-b0ec-4a53-9eea-223bc1709221)

See https://orcaui.umccr.org/runs/workflow/wfr.01JC1PQP5VA8AGYS94S1ZVYD1T as an example

**Comment by raylrui** [comment]
yes, currently "Resolved" State is only available for "Failed" State. If workflow failed, we can "resolve" it with comment.

Do we need to make it available for both "Failed" State and "Aborted" State ? 
@reisingerf

**Comment by raylrui** [comment]
yes, bug should be fixed after https://github.com/umccr/orca-ui/pull/59

**Comment by raylrui** [comment]
issue has been fixed
